sections/executive-team-section/index.js

```diff
@@ -14,10 +14,10 @@ import { people } from './people'
 const { h2 } = Heading.VARIANT
 
 const StyledSection = styled.section`
-  padding: ${rem('100px')} 0;
+  padding: ${rem(140)} 0 ${rem(140 - 84)};
```

Comment:
Hey Chris, `rem` does accept strings or numbers, but would it be better to keep consistency among the file/project?

Reply:
That's a really good question. I guess one benefit to using numbers is the ability to do calculations. But consistency is important as well so we need to decide if we should switch and, if so, how we go about it (change a file if we are working on it for other reasons or as a single dedicated task).

Reply:
I think this is something we can address in a different "consistency" issue

Reply:
@chrisnottca what does the 84 mean? apart from the consistency issue, I think we should try to not use "magic" numbers. If there are specific numbers like this, let's make variables to improve code readability.